### DISCUSSION TITLE
will you provide deprecation documentation for v7?

### DISCUSSION THREAD

**Discussion by arderyp** [deployphp/deployer#2731] [General]
I see you all very active on developing v7, which is awesome!

I was just wondering if you all plan to release deprecation documentation for those of us wanting to upgrade from v6 to v7 after the official release drops.

**Comment by antonmedv** [comment]
Already https://deployer.org/docs/7.x/UPGRADE

**Reply by antonmedv** [reply]
Btw, consider supporting my work on github to motivate me even more ;)

**Reply by arderyp** [reply]
Amazing, and will do!
